src/Utilities/Wrapper.js

```diff
@@ -1,25 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { usePermissions } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
-import { Spinner } from '@patternfly/react-core';
+import { usePermissionsWithContext } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
 
 const RenderWrapper = ({ cmp: Component, hideLoader, isRbacEnabled, inventoryRef, store, ...props }) => {
```

Comment:
Can you remove `hideLoader` from here and the propTypes since it is nowhere used anymore?

Reply:
Thanks for catching that! ...Odd that eslint did not notice it...